digital-ocean/container-linux/kubernetes/bootkube.tf

```diff
@@ -1,12 +1,18 @@
 # Self-hosted Kubernetes assets (kubeconfig, manifests)
 module "bootkube" {
-  source = "git::https://github.com/poseidon/terraform-render-bootkube.git?ref=b96d641f6d42cf5d9bf3ac36f557aa21cc157680"
+  source = "github.com/kinvolk/terraform-render-bootkube?ref=f5e41c83ec01e00187cb5c50cb7e9ec8275bd43e"
+
+  cluster_name = "${var.cluster_name}"
+  api_servers  = ["${format("%s.%s", var.cluster_name, var.dns_zone)}"]
+  etcd_servers = "${digitalocean_record.etcds.*.fqdn}"
+  asset_dir    = "${var.asset_dir}"
+
+  networking = "${var.networking}"
+
+  # only effective with Calico networking
+  network_encapsulation = "vxlan"
+  network_mtu           = "1450"
```

Comment:
Here we use another magic value than what is used for aws? Can you please elaborate how this was selected?

Reply:
Added from https://github.com/poseidon/typhoon/commit/147c21a4bd69fb92e763f99ea4bb5738bdd1af65

Reply:
Thanks!